dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -110,15 +121,21 @@ internal class DatadogRumMonitor(
         )
     }
 
-    fun addResourceTiming(key: Any, timing: RumEventData.Resource.Timing) {
+    internal fun addResourceTiming(key: Any, timing: RumEventData.Resource.Timing) {
         handleEvent(
             RumRawEvent.AddResourceTiming(key, timing)
         )
     }
 
-    private fun handleEvent(event: RumRawEvent) {
+    internal fun handleEvent(event: RumRawEvent) {
+        handler.removeCallbacks(keepAliveRunnable)
```

Comment:
I think we should try to stop this if we stop the feature. Maybe it will auto - stop when the handler will be GC but we need to make sure that is not keeping any reference alive to the DatadogRumMonitor.

Reply:
True, I'll add something to this effect